pixl_dcmd/src/pixl_dcmd/main.py

```diff
@@ -102,6 +102,22 @@ def anonymise_dicom(dataset: Dataset) -> Dataset:
     return dataset
 
 
+def get_project_name_as_string(dataset: Dataset) -> str:
```

Comment:
Agreed. Can use existing `_deid_helpers.py`. Or since it's not really de-id, and no other module fits, a new one called `_tag_helpers.py`? OR rename `_deid_helpers.py` to `_helpers.py` and put it there.

Reply:
I'd be tempted to make a new one for `_tag_helpers` or ` _dicom_helpers`?